notebooks_tsqr/NightLog.ipynb

```diff
@@ -72,8 +72,9 @@
     "usdf = 'https://usdf-rsp-dev.slac.stanford.edu'\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #48.                    del rec[f]
</pre><p>Actually, I think you might get an error here when you delete from the list you are looping over? Unsure in this instance but I think it is dangerous to do.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_logging_and_reporting/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I agree its dangerous to delete from the list being looped over.  But that's not what is happening here.  The delete is deleting one element from a dictionary (not a rec). The dictionary is not being looped over.  In fact, your concern is why the nukefields is calculated, then iterated over.  It avoid the naive approach of iterating directly over rec.keys(), which would be the situation you warn against.</p>